backend/sentenceTokenizer.py

```diff
@@ -0,0 +1,52 @@
+class Tokenizer:
+
+    def __init__(self, text):
+        self.text = text
+        import nltk
+        self.nltk = nltk
+        from nltk.corpus import stopwords
+        nltk.download('stopwords')
+        self.stopwords = set(stopwords.words('english'))
+        nltk.download('punkt')
+        nltk.download('averaged_perceptron_tagger')
+        self.punctuation = nltk.RegexpTokenizer(r"\w+")
+
+    def setText(self, newText):
+        self.text = newText
+
+    def addToText(self, newText):
```

Comment:
This method appends to the existing text but does not include a delimiter like a space or comma. Could that cause issues?

Reply:
Great suggestion! I will add a delimiter for this function.